Formula/w/wcstools.rb

```diff
@@ -0,0 +1,19 @@
+class Wcstools < Formula
+  desc "Tools for using World Coordinate Systems (WCS) in astronomical images"
+  homepage "http://tdc-www.harvard.edu/wcstools/"
+  url "http://tdc-www.harvard.edu/software/wcstools/wcstools-3.9.7.tar.gz"
+  sha256 "525f6970eb818f822db75c1526b3122b1af078affa572dce303de37df5c7b088"
+  license "LGPL-2.0-only"
```

Comment:
This doesn't seem accurate, here's a copyright header from the source code (`wcshead.c`):
```
This program is free software; you can redistribute it and/or
   modify it under the terms of the GNU General Public License
   as published by the Free Software Foundation; either version 2
   of the License, or (at your option) any later version.
```
So, shouldn't it be `GPL-2.0-or-later`?

Reply:
Modified to `GPL-2.0-or-later`